backend/app/utils/configs/validations.py

```
@@ -141,10 +150,35 @@ def validate_config_list(config_definition_key: str, page: int, page_size: int)
+    indexes = config_definition.get("indexes", [])
+
+    sortable_fields = {"config_key", "created_at", "modified_at", *indexes}
+    query_fields = {"page", "limit", "sort_by", "sort_order", "search"}
+
+    filterable_fields = {
+        field
+        for field in {*indexes, *sortable_fields, *query_fields}
+        if field != "config_key"
+    }
+
+    for key in request.query_params.keys():
```

Ruff (0.8.0)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)